src/views/catalog/utils/useWizardVmCreate.ts

```diff
@@ -43,6 +45,42 @@ export const useWizardVmCreate = (): UseWizardVmCreateValues => {
           vmDraft.spec.template.spec.domain.devices.disks.push(sysprepDisk());
           vmDraft.spec.template.spec.volumes.push(sysprepVolume(vmDraft));
         }
+
+        if (sshKey) {
+          // check that cloudInitConfigDrive exists
+          const cloudInitConfigDrive = vmDraft.spec.template.spec.volumes.some(
+            (v) => v.cloudInitConfigDrive,
+          );
+          if (!cloudInitConfigDrive) {
+            vmDraft.spec.template.spec.volumes = [
+              {
+                name:
+                  vmDraft.spec.template.spec.volumes.find((v) => v.cloudInitNoCloud)?.name ||
+                  'cloudinitdisk',
```

Comment:
```suggestion
                  'cloudinitdisk',
```
name will always be cloudinitdisk